explorer/services.go

```diff
@@ -41,16 +42,19 @@ func NewLocalServices(datalake DataLake, uuid string) *LocalServices {
 
 // NewRemoteServices initializes a services struct with a remote endpoint
 func NewRemoteServices(addr string, auth []ranger.ClientPlugin) (*Services, error) {
-	client := ranger.DefaultHttpClient()
```

Comment:
This is way too much work, we should create a ranger.Client plugin that set the http client. Then we can just pass a list